instance/models/mixins/openedx_database.py

```diff
@@ -95,6 +95,10 @@ def mysql_databases(self):
                     }
                 ],
             },
+            {
+                "name": self._get_mysql_database_name("programs"),
+                "user": self._get_mysql_user_name("program"),
+            },
```

Comment:
Nice improvement, will make adding new databases in the future easier.

> do we need the mysql_provisioned flag anymore?

I don't think we do. The methods that deprovision databases already seem to be designed so that they won't fail if the database or user does not exist, so `mysql_provisoined` flag doesn't make much sense anymore.